pkg/cmd/roachtest/versionupgrade.go

```diff
@@ -93,13 +93,16 @@ func runVersionUpgrade(ctx context.Context, t *test, c *cluster, buildVersion ve
 	if false {
 		// The version to create/update the fixture for. Must be released (i.e.
 		// can download it from the homepage); if that is not the case use the
-		// empty string which uses the local cockroach binary.
-		newV := "21.1.0"
-		predV, err := PredecessorVersion(*version.MustParse("v" + newV))
-		if err != nil {
-			t.Fatal(err)
-		}
-		makeVersionFixtureAndFatal(ctx, t, c, predV, newV)
+		// empty string which uses the local cockroach binary. Make sure that
+		// this binary then has the correct version. For example, to make a
+		// "v20.2" fixture, you will need a binary that has "v20.2" in the
+		// output of `./cockroach version`, and this process will end up
+		// creating fixtures that have "v20.2" in them. This would be part
+		// of tagging the master branch as v21.1 in the process of going
+		// through the major release for v20.2.
+		var makeFixtureVersion string
+		// makeFixtureVersion = "21.2.0"
```

Comment:
nit: do we still need this?

Reply:
We do - this should be the common case. I clarified.